src/Integration/Binding/BindingController.cs

```diff
@@ -147,8 +147,7 @@ void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
                     Strings.Bind_FirstTimeBinding :
                     Strings.Bind_UpdatingNewStyleBinding);
 
-
-            var solutionBindingOp = new SolutionBindingOperation(host);
+            var solutionBindingOp = new SolutionBindingOperation();
```

Comment:
Now that the `SolutionBindingOperation` has an empty constructor, it doesn't need to be created here.
I'll move it in a separate PR.